NoPythonDemo/StyleKitSample/AppDelegate.swift

```diff
@@ -16,6 +16,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
         // Override point for customization after application launch.
+        do {
```

Comment:
break this do/catch out into a separate method with a name that reflects what it does.